<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>boutiques</class>
 <widget class="QWidget" name="boutiques">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>683</width>
    <height>443</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>40</y>
     <width>521</width>
     <height>331</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Ajouter un boutique</string>
    </attribute>
    <widget class="QLabel" name="Nboutique">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>70</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Nom de boutique</string>
     </property>
    </widget>
    <widget class="QLabel" name="Cboutique">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>120</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Categorie</string>
     </property>
    </widget>
    <widget class="QLabel" name="Sboutique">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>170</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Stand</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_nb">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>70</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_cb">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>120</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_sb">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>170</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_ajouter">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>260</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Ajouter</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_annuler">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>260</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Retour</string>
     </property>
    </widget>
    <widget class="QTabWidget" name="tabWidget_2">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>160</y>
       <width>521</width>
       <height>331</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Ajouter un boutique</string>
      </attribute>
      <widget class="QLabel" name="Nboutique_2">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>70</y>
         <width>91</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Nom de boutique</string>
       </property>
      </widget>
      <widget class="QLabel" name="Cboutique_2">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>120</y>
         <width>101</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Categorie</string>
       </property>
      </widget>
      <widget class="QLabel" name="Sboutique_2">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>170</y>
         <width>81</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Stand</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_nb_2">
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>70</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_cb_2">
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>120</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_sb_2">
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>170</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pb_ajouter_2">
       <property name="geometry">
        <rect>
         <x>400</x>
         <y>260</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Ajouter</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pb_annuler_2">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>260</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Retour</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_4">
      <attribute name="title">
       <string>Afficher les boutiques</string>
      </attribute>
      <widget class="QTableView" name="tabetudiant_2">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>40</y>
         <width>331</width>
         <height>231</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Afficher les boutiques</string>
    </attribute>
    <widget class="QTableView" name="tabetudiant">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>40</y>
       <width>331</width>
       <height>231</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
